pruning/src/lib.rs

```diff
@@ -153,43 +325,36 @@ impl PruningSeed {
         } else if blockchain_height > CRYPTONOTE_MAX_BLOCK_NUMBER {
             Err(PruningError::BlockChainHeightTooLarge)
         } else {
```

Comment:
Could early return here to prevent this `else` block being indented:
```rust
if block_height > CRYPTONOTE_MAX_BLOCK_NUMBER || block_height > blockchain_height {
    return Err(PruningError::BlockHeightTooLarge);
}

if blockchain_height > CRYPTONOTE_MAX_BLOCK_NUMBER {
    return Err(PruningError::BlockChainHeightTooLarge);
}

// rest of function
```